src/http/apis/bankid/utils.ts

```diff
@@ -0,0 +1,32 @@
+import Configuration from "../../../config/index";
```

Comment:
I don't think these utility functions create enough abstractions for what they do. Having the logic closer to the actual API-functionality renders, IMO, more readable code and code that's easier to maintain.

Reply:
I see what you mean, although I don't want to keep all api logic for bank id in a single file.

After thinking about it I think that the best way would be to separate all api logic for bank id into three files: auth, collect and cancel. Each of these files include their own separate _util functions_.